pkg/logql/engine.go

```diff
@@ -118,12 +118,17 @@ type EngineOpts struct {
 	// MaxLookBackPeriod is the maximum amount of time to look back for log lines.
 	// only used for instant log queries.
 	MaxLookBackPeriod time.Duration `yaml:"max_look_back_period"`
+
+	// LogExecutingQuery will control if we log the query name when Exec is called.
```

Comment:
It's not just the name, right?

```suggestion
	// LogExecutingQuery will control if we log the query when Exec is called.
```